.buildkite/test-check/scripts/test-check-xpu.sh

```
@@ -0,0 +1,31 @@
+
+docker run -tid --disable-content-trust --privileged --name "$CONTAINER_NAME" -v "$REPO_PATH:/workspace" -w /workspace "$IMAGE_NAME"
+echo "Show the container list after docker run ... "
+docker ps -a | grep "$CONTAINER_NAME"
+
+echo "Setup test environment..."
+docker exec $CONTAINER_NAME \
+              bash -c "uv pip install --upgrade pip setuptools && \
+              uv pip install .[dev] --extra-index-url https://download.pytorch.org/whl/xpu"
+
+echo "Run tests inside the container..."
+docker exec -e NUMA_NODE=${NUMA_NODE} -e NUMA_CPUSET=${NUMA_CPUSET} -e ZE_AFFINITY_MASK=${ZE_AFFINITY_MASK} $CONTAINER_NAME \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)